coverage.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+cargo llvm-cov nextest -j 10 --lcov --output-path lcov.info
```

Comment:
are we using this file for anything at present?

Reply:
The [Coverage Gutters](https://marketplace.visualstudio.com/items?itemName=ryanluker.vscode-coverage-gutters) vscode extension will pick it up if it exists.